### DISCUSSION TITLE
Thermostat with possibility to send % valve opening

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#15778] [Devices]
Hi there, i am looking for a replacment of TS0601_TZE200_cwnjrr72 which possibility to send % of valve opening instead of only open/close commands. Would you share your experience?

**Comment by lux73** [comment]
Eurotronic Spirit have this 🙂

**Comment by stefan-oskamp** [comment]
Thermally activated floor heating valve actuators are cheap, they can be used on valves for thermostats, and, if you drive them with a dimmer module, you should get what you are looking for. Not sure, though, whether this makes a lot of sense.

**Comment by Ivano62** [comment]
Moes [BRT-100-TRV] also have this

**Comment by mipoat** [comment]
Are you @Ivano62 sure the Moes can do that?
According to https://www.zigbee2mqtt.io/devices/BRT-100-TRV.html 
_Position. Value can be found in the published state on the position property. It's not possible to read (/get) or write (/set) this value. The unit of this value is %_

**Comment by Ivano62** [comment]
With the latest versions it works fine. Thanks
